
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Relevant Publications &#8212; Computational Protein Design Minicourse</title>
    
  <link rel="stylesheet" href="_static/css/index.f658d18f9b420779cfdf24aa0a7e2d77.css">

    
  <link rel="stylesheet"
    href="_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    
      
  <link rel="stylesheet"
    href="_static/vendor/open-sans_all/1.44.1/index.css">
  <link rel="stylesheet"
    href="_static/vendor/lato_latin-ext/1.44.1/index.css">

    
    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    <link rel="stylesheet" href="_static/sphinx-book-theme.e7340bb3dbd8dde6db86f25597f54a1b.css" type="text/css" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="_static/panels-main.c949a650a448cc0ae9fd3441c0e17fb0.css" />
    <link rel="stylesheet" type="text/css" href="_static/panels-variables.06eb56fa6e07937060861dad626602ad.css" />
    
  <link rel="preload" as="script" href="_static/js/index.d3f166471bb80abb5163.js">

    <script id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/togglebutton.js"></script>
    <script src="_static/clipboard.min.js"></script>
    <script src="_static/copybutton.js"></script>
    <script >var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="_static/sphinx-book-theme.7d483ff0a819d6edff12ce0b1ead3928.js"></script>
    <script async="async" src="https://unpkg.com/thebelab@latest/lib/index.js"></script>
    <script >
        const thebe_selector = ".thebe"
        const thebe_selector_input = "pre"
        const thebe_selector_output = ".output"
    </script>
    <script async="async" src="_static/sphinx-thebe.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="prev" title="Course Schedule" href="schedule.html" />

    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="en" />



  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    

    <div class="container-xl">
      <div class="row">
          
<div class="col-12 col-md-3 bd-sidebar site-navigation show" id="site-navigation">
    
        <div class="navbar-brand-box">
<a class="navbar-brand text-wrap" href="index.html">
  
  <img src="_static/logo.png" class="logo" alt="logo">
  
  
  <h1 class="site-logo" id="site-title">Computational Protein Design Minicourse</h1>
  
</a>
</div><form class="bd-search d-flex align-items-center" action="search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form>
<nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
    <ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="installation_instructions.html">
   Installing Course Prerequisites
  </a>
 </li>
</ul>
<ul class="current nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="02_LigandDesign.html">
   Ligand Binding Site Design Exercise
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="01_RosettaDesign.html">
   Rosetta Design
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="00_getting_started_with_rosetta.html">
   Getting started with PyRosetta and PyMOL
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="accessing_the_course.html">
   Accessing the course
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="survey.html">
   Survey for choosing projects
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="ddg_exercise.html">
   ΔΔG Exercise
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="_intro.html">
   Homepage
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="schedule.html">
   Course Schedule
  </a>
 </li>
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   Relevant Publications
  </a>
 </li>
</ul>

</nav> <!-- To handle the deprecated key -->

<div class="navbar_extra_footer">
  Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
</div>

</div>


          


          
<main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
    
    <div class="topbar container-xl fixed-top">
    <div class="topbar-contents row">
        <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show"></div>
        <div class="col pl-md-4 topbar-main">
            
            <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
                data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
                aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
                title="Toggle navigation" data-toggle="tooltip" data-placement="left">
                <i class="fas fa-bars"></i>
                <i class="fas fa-arrow-left"></i>
                <i class="fas fa-arrow-up"></i>
            </button>
            
            
<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
            class="fas fa-download"></i></button>

    <div class="dropdown-buttons">
        <!-- ipynb file if we had a myst markdown file -->
        
        <!-- Download raw file -->
        <a class="dropdown-buttons" href="_sources/papers.md"><button type="button"
                class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
                data-placement="left">.md</button></a>
        <!-- Download PDF via print -->
        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
            onClick="window.print()" data-toggle="tooltip" data-placement="left">.pdf</button>
    </div>
</div>

            <!-- Source interaction buttons -->


            <!-- Full screen (wrap in <a> to have style consistency -->
            <a class="full-screen-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
                    data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
                    title="Fullscreen mode"><i
                        class="fas fa-expand"></i></button></a>

            <!-- Launch buttons -->

        </div>

        <!-- Table of contents -->
        <div class="d-none d-md-block col-md-2 bd-toc show">
            
        <div class="tocsection onthispage pt-5 pb-3">
            <i class="fas fa-list"></i>
            Contents
        </div>
        <nav id="bd-toc-nav">
            <ul class="nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#project-readings">
   Project Readings
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#project-1-predicting-effects-of-mutations-modeling-sars-cov-2-spike-variants-binding-to-ace2">
     Project 1: Predicting effects of mutations: modeling SARS-CoV-2 spike variants binding to Ace2
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#project-2-optimize-a-ligand-binding-site-to-control-gene-expression-in-mammalian-cells">
     Project 2: Optimize a ligand binding site to control gene expression in mammalian cells
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#project-3-de-novo-design-design-a-complete-protein-with-a-new-shape-from-scratch">
     Project 3: De novo design: Design a complete protein with a new shape from scratch
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#optional-readings">
     Optional Readings
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#week-1">
   Week 1
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#week-2">
   Week 2
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#week-3">
   Week 3
  </a>
 </li>
</ul>

        </nav>
        
        </div>
    </div>
</div>
    <div id="main-content" class="row">
        <div class="col-12 col-md-9 pl-md-3 pr-md-0">
        
              <div>
                
  <div class="section" id="relevant-publications">
<h1>Relevant Publications<a class="headerlink" href="#relevant-publications" title="Permalink to this headline">¶</a></h1>
<div class="section" id="project-readings">
<h2>Project Readings<a class="headerlink" href="#project-readings" title="Permalink to this headline">¶</a></h2>
<p>Below are readings that serve as good introductions for each of the projects:</p>
<div class="section" id="project-1-predicting-effects-of-mutations-modeling-sars-cov-2-spike-variants-binding-to-ace2">
<h3>Project 1: Predicting effects of mutations: modeling SARS-CoV-2 spike variants binding to Ace2<a class="headerlink" href="#project-1-predicting-effects-of-mutations-modeling-sars-cov-2-spike-variants-binding-to-ace2" title="Permalink to this headline">¶</a></h3>
<ul class="simple">
<li><p>Glasgow, A., Glasgow, J., Limonta, D., Solomon, P., Lui, I., Zhang, Y., Nix, M. A., Rettko, N. J., Zha, S., Yamin, R., Kao, K., Rosenberg, O. S., Ravetch, J. V., Wiita, A. P., Leung, K. K., Lim, S. A., Zhou, X. X., Hobman, T. C., Kortemme, T., &amp; Wells, J. A. (2020). <em><strong>Engineered ACE2 receptor traps potently neutralize SARS-CoV-2.</strong></em>  Proceedings of the National Academy of Sciences of the United States of America, 117(45), 28046–28055. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7668070/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1073/pnas.2016093117">Journal</a></p></li>
<li><p>Chan, K. K., Dorosky, D., Sharma, P., Abbasi, S. A., Dye, J. M., Kranz, D. M., Herbert, A. S., &amp; Procko, E. (2020). <em><strong>Engineering human ACE2 to optimize binding to the spike protein of SARS coronavirus 2.</strong></em> Science (Vol. 369, Issue 6508, pp. 1261–1265). | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7574912/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1126/science.abc0870">Journal</a></p></li>
<li><p>Starr, T. N., Greaney, A. J., Hilton, S. K., Ellis, D., Crawford, K. H. D., Dingens, A. S., Navarro, M. J., Bowen, J. E., Tortorici, M. A., Walls, A. C., King, N. P., Veesler, D., &amp; Bloom, J. D. (2020). <em><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding.</strong></em> Cell, 182(5), 1295–1310.e20. | <a class="reference external" href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmc7418704/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1016/j.cell.2020.08.012">Journal</a></p></li>
</ul>
</div>
<div class="section" id="project-2-optimize-a-ligand-binding-site-to-control-gene-expression-in-mammalian-cells">
<h3>Project 2: Optimize a ligand binding site to control gene expression in mammalian cells<a class="headerlink" href="#project-2-optimize-a-ligand-binding-site-to-control-gene-expression-in-mammalian-cells" title="Permalink to this headline">¶</a></h3>
<ul class="simple">
<li><p>Glasgow, A. A., Huang, Y.-M., Mandell, D. J., Thompson, M., Ritterson, R., Loshbaugh, A. L., Pellegrino, J., Krivacic, C., Pache, R. A., Barlow, K. A., Ollikainen, N., Jeon, D., Kelly, M. J. S., Fraser, J. S., &amp; Kortemme, T. (2019). <em><strong>Computational design of a modular protein sense-response system.</strong></em> Science, 366(6468), 1024–<a class="reference external" href="http://1028.ER">1028.ER</a> | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7343396/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1126/science.aax8780">Journal</a></p></li>
<li><p>Billas, I. M. L., Iwema, T., Garnier, J.-M., Mitschler, A., Rochel, N., &amp; Moras, D. (2003). <em><strong>Structural adaptability in the ligand-binding pocket of the ecdysone hormone receptor.</strong></em> Nature, 426(6962), 91–96. | <a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/14595375/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1038/nature02112">Journal</a></p></li>
</ul>
</div>
<div class="section" id="project-3-de-novo-design-design-a-complete-protein-with-a-new-shape-from-scratch">
<h3>Project 3: De novo design: Design a complete protein with a new shape from scratch<a class="headerlink" href="#project-3-de-novo-design-design-a-complete-protein-with-a-new-shape-from-scratch" title="Permalink to this headline">¶</a></h3>
<ul class="simple">
<li><p>Pan, X., Thompson, M. C., Zhang, Y., Liu, L., Fraser, J. S., Kelly, M. J. S., &amp; Kortemme, T. (2020). <em><strong>Expanding the space of protein geometries by computational design of de novo fold families.</strong></em> Science, 369(6507), 1132–1136. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7787817/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1126/science.abc0881">Journal</a></p></li>
<li><p>Rocklin, G. J., Chidyausiku, T. M., Goreshnik, I., Ford, A., Houliston, S., Lemak, A., Carter, L., Ravichandran, R., Mulligan, V. K., Chevalier, A., Arrowsmith, C. H., &amp; Baker, D. (2017). <em><strong>Global analysis of protein folding using massively parallel design, synthesis, and testing.</strong></em> Science, 357(6347), 168–175. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5568797/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1126/science.aan0693">Journal</a></p></li>
</ul>
</div>
<div class="section" id="optional-readings">
<h3>Optional Readings<a class="headerlink" href="#optional-readings" title="Permalink to this headline">¶</a></h3>
<p>Below are references and links for optional readings that supplement the course’s lectures and exercises.</p>
</div>
</div>
<div class="section" id="week-1">
<h2>Week 1<a class="headerlink" href="#week-1" title="Permalink to this headline">¶</a></h2>
<p>Session 1.1 (Mar 29): Introduction to course, topics, and projects</p>
<ul class="simple">
<li><p>Pan, X., &amp; Kortemme, T. (2021). <em><strong>Recent advances in de novo protein design: principles, methods, and applications.</strong></em> The Journal of Biological Chemistry, 100558. | <a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33744284/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1016/j.jbc.2021.100558">Journal</a></p></li>
</ul>
<p>Session 1.2 (Mar 30): Milestones in the field</p>
<ul class="simple">
<li><p>Paper TBD</p></li>
</ul>
<p>Session 1.3 (Mar 31): Design energy functions</p>
<ul class="simple">
<li><p>Park, H., Bradley, P., Greisen, P., Jr, Liu, Y., Mulligan, V. K., Kim, D. E., Baker, D., &amp; DiMaio, F. (2016). <em><strong>Simultaneous Optimization of Biomolecular Energy Functions on Features from Small Molecules and Macromolecules. Journal of Chemical Theory and Computation, 12(12), 6201–6212.</strong></em> | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515585/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1021/acs.jctc.6b00819">Journal</a></p></li>
<li><p>Alford, R. F., Leaver-Fay, A., Jeliazkov, J. R., O’Meara, M. J., DiMaio, F. P., Park, H., Shapovalov, M. V., Renfrew, P. D., Mulligan, V. K., Kappel, K., Labonte, J. W., Pacella, M. S., Bonneau, R., Bradley, P., Dunbrack, R. L., Jr, Das, R., Baker, D., Kuhlman, B., Kortemme, T., &amp; Gray, J. J. (2017). <em><strong>The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design.</strong></em> Journal of Chemical Theory and Computation, 13(6), 3031–3048. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5717763/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1021/acs.jctc.7b00125">Journal</a></p></li>
</ul>
<p>Session 1.4 (Apr 01): Sampling in design</p>
<ul class="simple">
<li><p>Mandell, D. J., &amp; Kortemme, T. (2009). <em><strong>Backbone flexibility in computational protein design.</strong></em> Current Opinion in Biotechnology, 20(4), 420–428. | <a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/19709874/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1016/j.copbio.2009.07.006">Journal</a></p></li>
</ul>
<p>Session 1.5 (Apr 02): De novo folds and filters</p>
<ul class="simple">
<li><p>Koga, N., Tatsumi-Koga, R., Liu, G., Xiao, R., Acton, T. B., Montelione, G. T., &amp; Baker, D. (2012). <em><strong>Principles for designing ideal protein structures.</strong></em> Nature, 491(7423), 222–227. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3705962/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1038/nature11600">Journal</a></p></li>
<li><p>Rocklin, G. J., Chidyausiku, T. M., Goreshnik, I., Ford, A., Houliston, S., Lemak, A., Carter, L., Ravichandran, R., Mulligan, V. K., Chevalier, A., Arrowsmith, C. H., &amp; Baker, D. (2017). <em><strong>Global analysis of protein folding using massively parallel design, synthesis, and testing.</strong></em> Science, 357(6347), 168–175. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5568797/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1126/science.aan0693">Journal</a></p></li>
</ul>
</div>
<div class="section" id="week-2">
<h2>Week 2<a class="headerlink" href="#week-2" title="Permalink to this headline">¶</a></h2>
<p>Session 2.1 (Apr 05): Function design</p>
<ul class="simple">
<li><p>Fleishman, S. J., &amp; Baker, D. (2012). <em><strong>Role of the biomolecular energy gap in protein design, structure, and evolution.</strong></em> Cell, 149(2), 262–273. | <a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/22500796/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1016/j.cell.2012.03.016">Journal</a></p></li>
</ul>
<p>Session 2.2 (Apr 06): Teams present / discuss proposals</p>
<ul class="simple">
<li><p>No reading</p></li>
</ul>
<p>Session 2.3 (Apr 07): Structure prediction</p>
<ul class="simple">
<li><p>Yang, J., Anishchenko, I., Park, H., Peng, Z., Ovchinnikov, S., &amp; Baker, D. (2020). <em><strong>Improved protein structure prediction using predicted interresidue orientations.</strong></em> Proceedings of the National Academy of Sciences of the United States of America, 117(3), 1496–1503. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6983395/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1073/pnas.1914677117">Journal</a></p></li>
</ul>
<p>Session 2.4 (Apr 08): Loop modeling and filters</p>
<ul class="simple">
<li><p>Mandell, D. J., Coutsias, E. A., &amp; Kortemme, T. (2009). <em><strong>Sub-angstrom accuracy in protein loop reconstruction by robotics-inspired conformational sampling.</strong></em> Nature Methods, 6(8), 551–552. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847683/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1038/nmeth0809-551">Journal</a></p></li>
<li><p>Rocklin, G. J., Chidyausiku, T. M., Goreshnik, I., Ford, A., Houliston, S., Lemak, A., Carter, L., Ravichandran, R., Mulligan, V. K., Chevalier, A., Arrowsmith, C. H., &amp; Baker, D. (2017). <em><strong>Global analysis of protein folding using massively parallel design, synthesis, and testing.</strong></em> Science, 357(6347), 168–175. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5568797/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1126/science.aan0693">Journal</a></p></li>
</ul>
<p>Session 2.5 (Apr 09): Teams discuss project with TAs</p>
<ul class="simple">
<li><p>No paper</p></li>
</ul>
</div>
<div class="section" id="week-3">
<h2>Week 3<a class="headerlink" href="#week-3" title="Permalink to this headline">¶</a></h2>
<p>Session 3.1 (Apr 12): Teams present progress</p>
<ul class="simple">
<li><p>No paper</p></li>
</ul>
<p>Session 3.2 (Apr 13): Protein dynamics</p>
<ul class="simple">
<li><p>Broom, A., Rakotoharisoa, R. V., Thompson, M. C., Zarifi, N., Nguyen, E., Mukhametzhanov, N., Liu, L., Fraser, J. S., &amp; Chica, R. A. (2020). <em><strong>Ensemble-based enzyme design can recapitulate the effects of laboratory directed evolution in silico.</strong></em> Nature Communications, 11(1), 4808. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7511930/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1038/s41467-020-18619-x">Journal</a></p></li>
</ul>
<p>Session 3.3 (Apr 14): Design in disease applications</p>
<ul class="simple">
<li><p>Stein, A., Fowler, D. M., Hartmann-Petersen, R., &amp; Lindorff-Larsen, K. (2019). <em><strong>Biophysical and Mechanistic Models for Disease-Causing Protein Variants.</strong></em> Trends in Biochemical Sciences, 44(7), 575–588.  | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6579676/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1016/j.tibs.2019.01.003">Journal</a></p></li>
</ul>
<p>Session 3.4 (Apr 15): Limitations of design</p>
<ul class="simple">
<li><p>Baker, D. (2019). <em><strong>What has de novo protein design taught us about protein folding and biophysics?</strong></em> Protein Science: A Publication of the Protein Society, 28(4), 678–683. | <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6423711/">Pubmed</a> | <a class="reference external" href="https://doi.org/10.1002/pro.3588">Journal</a></p></li>
</ul>
<p>Session 3.5 (Apr 16): Teams present</p>
<ul class="simple">
<li><p>No paper</p></li>
</ul>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
        
        <div class='prev-next-bottom'>
            
    <a class='left-prev' id="prev-link" href="schedule.html" title="previous page">Course Schedule</a>

        </div>
        
        </div>
    </div>
    <footer class="footer mt-5 mt-md-0">
    <div class="container">
      <p>
        
          By Kortemme Lab<br/>
        
            &copy; Copyright 2021.<br/>
      </p>
    </div>
  </footer>
</main>


      </div>
    </div>

    
  <script src="_static/js/index.d3f166471bb80abb5163.js"></script>


    
  </body>
</html>