# Project Info

The slides introducing each project can be found in the Lectures Box folder at: https://ucsf.box.com/s/ajwi3yl08deslzr9rgouwjsoi0j1p083

## Project Groups and Readings

### Group 1: Predicting effects of mutations: modeling SARS-CoV-2 Spike variants binding to ACE2.

- Upneet Kaur
- Kevin Nie
- Chris Chang
- Maxine Bi
- Ben Orr
- TA: Chris Mathy

Meeting place: sidechain salon

Papers:
- Glasgow, A., Glasgow, J., Limonta, D., Solomon, P., Lui, I., Zhang, Y., Nix, M. A., Rettko, N. J., Zha, S., Yamin, R., Kao, K., Rosenberg, O. S., Ravetch, J. V., Wiita, A. P., Leung, K. K., Lim, S. A., Zhou, X. X., Hobman, T. C., Kortemme, T., & Wells, J. A. (2020). ***Engineered ACE2 receptor traps potently neutralize SARS-CoV-2.***  Proceedings of the National Academy of Sciences of the United States of America, 117(45), 28046–28055. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7668070/) | [Journal](https://doi.org/10.1073/pnas.2016093117)
- Chan, K. K., Dorosky, D., Sharma, P., Abbasi, S. A., Dye, J. M., Kranz, D. M., Herbert, A. S., & Procko, E. (2020). ***Engineering human ACE2 to optimize binding to the spike protein of SARS coronavirus 2.*** Science (Vol. 369, Issue 6508, pp. 1261–1265). | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7574912/) | [Journal](https://doi.org/10.1126/science.abc0870)
- Starr, T. N., Greaney, A. J., Hilton, S. K., Ellis, D., Crawford, K. H. D., Dingens, A. S., Navarro, M. J., Bowen, J. E., Tortorici, M. A., Walls, A. C., King, N. P., Veesler, D., & Bloom, J. D. (2020). ***Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding.*** Cell, 182(5), 1295–1310.e20. | [Pubmed](http://www.ncbi.nlm.nih.gov/pmc/articles/pmc7418704/) | [Journal](https://doi.org/10.1016/j.cell.2020.08.012)

### Group 2: Optimize a ligand binding site to control gene expression in mammalian cells,

- Dominic Grisingher
- Sunny Choi
- Siyi Wang
- Mai Nobuhara
- Simone Harrison
- TA: Simon Kretschmer

Meeting place: peptide parlor

Papers:
- Glasgow, A. A., Huang, Y.-M., Mandell, D. J., Thompson, M., Ritterson, R., Loshbaugh, A. L., Pellegrino, J., Krivacic, C., Pache, R. A., Barlow, K. A., Ollikainen, N., Jeon, D., Kelly, M. J. S., Fraser, J. S., & Kortemme, T. (2019). ***Computational design of a modular protein sense-response system.*** Science, 366(6468), 1024–1028.ER | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7343396/) | [Journal](https://doi.org/10.1126/science.aax8780)
- Billas, I. M. L., Iwema, T., Garnier, J.-M., Mitschler, A., Rochel, N., & Moras, D. (2003). ***Structural adaptability in the ligand-binding pocket of the ecdysone hormone receptor.*** Nature, 426(6962), 91–96. | [Pubmed](https://pubmed.ncbi.nlm.nih.gov/14595375/) | [Journal](https://doi.org/10.1038/nature02112)


### Group 3: De novo design: Design a complete protein with a new shape from scratch using an established method (LUCS), and explore designing a de novo ligand binding site.
- Cole Bracken
- Soumya Remesh
- Daphne Chen
- Brenda Miao
- Rian Kormos
- TA: Cody Krivacic

Meeting place: de novo dwelling

Papers:
- Pan, X., Thompson, M. C., Zhang, Y., Liu, L., Fraser, J. S., Kelly, M. J. S., & Kortemme, T. (2020). ***Expanding the space of protein geometries by computational design of de novo fold families.*** Science, 369(6507), 1132–1136. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7787817/) | [Journal](https://doi.org/10.1126/science.abc0881)
- Rocklin, G. J., Chidyausiku, T. M., Goreshnik, I., Ford, A., Houliston, S., Lemak, A., Carter, L., Ravichandran, R., Mulligan, V. K., Chevalier, A., Arrowsmith, C. H., & Baker, D. (2017). ***Global analysis of protein folding using massively parallel design, synthesis, and testing.*** Science, 357(6347), 168–175. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5568797/) | [Journal](https://doi.org/10.1126/science.aan0693)
