# Relevant Publications

## Project Readings

Below are readings that serve as good introductions for each of the projects:

### Project 1: Predicting effects of mutations: modeling SARS-CoV-2 spike variants binding to Ace2
- Glasgow, A., Glasgow, J., Limonta, D., Solomon, P., Lui, I., Zhang, Y., Nix, M. A., Rettko, N. J., Zha, S., Yamin, R., Kao, K., Rosenberg, O. S., Ravetch, J. V., Wiita, A. P., Leung, K. K., Lim, S. A., Zhou, X. X., Hobman, T. C., Kortemme, T., & Wells, J. A. (2020). ***Engineered ACE2 receptor traps potently neutralize SARS-CoV-2.***  Proceedings of the National Academy of Sciences of the United States of America, 117(45), 28046–28055. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7668070/) | [Journal](https://doi.org/10.1073/pnas.2016093117)
- Chan, K. K., Dorosky, D., Sharma, P., Abbasi, S. A., Dye, J. M., Kranz, D. M., Herbert, A. S., & Procko, E. (2020). ***Engineering human ACE2 to optimize binding to the spike protein of SARS coronavirus 2.*** Science (Vol. 369, Issue 6508, pp. 1261–1265). | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7574912/) | [Journal](https://doi.org/10.1126/science.abc0870)
- Starr, T. N., Greaney, A. J., Hilton, S. K., Ellis, D., Crawford, K. H. D., Dingens, A. S., Navarro, M. J., Bowen, J. E., Tortorici, M. A., Walls, A. C., King, N. P., Veesler, D., & Bloom, J. D. (2020). ***Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding.*** Cell, 182(5), 1295–1310.e20. | [Pubmed](http://www.ncbi.nlm.nih.gov/pmc/articles/pmc7418704/) | [Journal](https://doi.org/10.1016/j.cell.2020.08.012)

### Project 2: Optimize a ligand binding site to control gene expression in mammalian cells
- Glasgow, A. A., Huang, Y.-M., Mandell, D. J., Thompson, M., Ritterson, R., Loshbaugh, A. L., Pellegrino, J., Krivacic, C., Pache, R. A., Barlow, K. A., Ollikainen, N., Jeon, D., Kelly, M. J. S., Fraser, J. S., & Kortemme, T. (2019). ***Computational design of a modular protein sense-response system.*** Science, 366(6468), 1024–1028.ER | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7343396/) | [Journal](https://doi.org/10.1126/science.aax8780)
- Billas, I. M. L., Iwema, T., Garnier, J.-M., Mitschler, A., Rochel, N., & Moras, D. (2003). ***Structural adaptability in the ligand-binding pocket of the ecdysone hormone receptor.*** Nature, 426(6962), 91–96. | [Pubmed](https://pubmed.ncbi.nlm.nih.gov/14595375/) | [Journal](https://doi.org/10.1038/nature02112)

### Project 3: De novo design: Design a complete protein with a new shape from scratch
- Pan, X., Thompson, M. C., Zhang, Y., Liu, L., Fraser, J. S., Kelly, M. J. S., & Kortemme, T. (2020). ***Expanding the space of protein geometries by computational design of de novo fold families.*** Science, 369(6507), 1132–1136. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7787817/) | [Journal](https://doi.org/10.1126/science.abc0881)
- Rocklin, G. J., Chidyausiku, T. M., Goreshnik, I., Ford, A., Houliston, S., Lemak, A., Carter, L., Ravichandran, R., Mulligan, V. K., Chevalier, A., Arrowsmith, C. H., & Baker, D. (2017). ***Global analysis of protein folding using massively parallel design, synthesis, and testing.*** Science, 357(6347), 168–175. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5568797/) | [Journal](https://doi.org/10.1126/science.aan0693)

### Optional Readings

Below are references and links for optional readings that supplement the course's lectures and exercises. 

## Week 1
Session 1.1 (Mar 29): Introduction to course, topics, and projects
- Pan, X., & Kortemme, T. (2021). ***Recent advances in de novo protein design: principles, methods, and applications.*** The Journal of Biological Chemistry, 100558. | [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33744284/) | [Journal](https://doi.org/10.1016/j.jbc.2021.100558)

Session 1.2 (Mar 30): Milestones in the field
- Paper TBD

Session 1.3 (Mar 31): Design energy functions
- Park, H., Bradley, P., Greisen, P., Jr, Liu, Y., Mulligan, V. K., Kim, D. E., Baker, D., & DiMaio, F. (2016). ***Simultaneous Optimization of Biomolecular Energy Functions on Features from Small Molecules and Macromolecules. Journal of Chemical Theory and Computation, 12(12), 6201–6212.*** | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515585/) | [Journal](https://doi.org/10.1021/acs.jctc.6b00819)
- Alford, R. F., Leaver-Fay, A., Jeliazkov, J. R., O’Meara, M. J., DiMaio, F. P., Park, H., Shapovalov, M. V., Renfrew, P. D., Mulligan, V. K., Kappel, K., Labonte, J. W., Pacella, M. S., Bonneau, R., Bradley, P., Dunbrack, R. L., Jr, Das, R., Baker, D., Kuhlman, B., Kortemme, T., & Gray, J. J. (2017). ***The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design.*** Journal of Chemical Theory and Computation, 13(6), 3031–3048. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5717763/) | [Journal](https://doi.org/10.1021/acs.jctc.7b00125) 

Session 1.4 (Apr 01): Sampling in design
- Mandell, D. J., & Kortemme, T. (2009). ***Backbone flexibility in computational protein design.*** Current Opinion in Biotechnology, 20(4), 420–428. | [Pubmed](https://pubmed.ncbi.nlm.nih.gov/19709874/) | [Journal](https://doi.org/10.1016/j.copbio.2009.07.006) 

Session 1.5 (Apr 02): De novo folds and filters
- Koga, N., Tatsumi-Koga, R., Liu, G., Xiao, R., Acton, T. B., Montelione, G. T., & Baker, D. (2012). ***Principles for designing ideal protein structures.*** Nature, 491(7423), 222–227. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3705962/) | [Journal](https://doi.org/10.1038/nature11600) 
- Rocklin, G. J., Chidyausiku, T. M., Goreshnik, I., Ford, A., Houliston, S., Lemak, A., Carter, L., Ravichandran, R., Mulligan, V. K., Chevalier, A., Arrowsmith, C. H., & Baker, D. (2017). ***Global analysis of protein folding using massively parallel design, synthesis, and testing.*** Science, 357(6347), 168–175. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5568797/) | [Journal](https://doi.org/10.1126/science.aan0693)

## Week 2
Session 2.1 (Apr 05): Function design
- Fleishman, S. J., & Baker, D. (2012). ***Role of the biomolecular energy gap in protein design, structure, and evolution.*** Cell, 149(2), 262–273. | [Pubmed](https://pubmed.ncbi.nlm.nih.gov/22500796/) | [Journal](https://doi.org/10.1016/j.cell.2012.03.016)

Session 2.2 (Apr 06): Teams present / discuss proposals
- No reading

Session 2.3 (Apr 07): Structure prediction
- Yang, J., Anishchenko, I., Park, H., Peng, Z., Ovchinnikov, S., & Baker, D. (2020). ***Improved protein structure prediction using predicted interresidue orientations.*** Proceedings of the National Academy of Sciences of the United States of America, 117(3), 1496–1503. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6983395/) | [Journal](https://doi.org/10.1073/pnas.1914677117) 

Session 2.4 (Apr 08): Loop modeling and filters
- Mandell, D. J., Coutsias, E. A., & Kortemme, T. (2009). ***Sub-angstrom accuracy in protein loop reconstruction by robotics-inspired conformational sampling.*** Nature Methods, 6(8), 551–552. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847683/) | [Journal](https://doi.org/10.1038/nmeth0809-551)
- Rocklin, G. J., Chidyausiku, T. M., Goreshnik, I., Ford, A., Houliston, S., Lemak, A., Carter, L., Ravichandran, R., Mulligan, V. K., Chevalier, A., Arrowsmith, C. H., & Baker, D. (2017). ***Global analysis of protein folding using massively parallel design, synthesis, and testing.*** Science, 357(6347), 168–175. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5568797/) | [Journal](https://doi.org/10.1126/science.aan0693)

Session 2.5 (Apr 09): Teams discuss project with TAs
- No paper

## Week 3
Session 3.1 (Apr 12): Teams present progress
- No paper

Session 3.2 (Apr 13): Protein dynamics
- Broom, A., Rakotoharisoa, R. V., Thompson, M. C., Zarifi, N., Nguyen, E., Mukhametzhanov, N., Liu, L., Fraser, J. S., & Chica, R. A. (2020). ***Ensemble-based enzyme design can recapitulate the effects of laboratory directed evolution in silico.*** Nature Communications, 11(1), 4808. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7511930/) | [Journal](https://doi.org/10.1038/s41467-020-18619-x)

Session 3.3 (Apr 14): Design in disease applications
- Stein, A., Fowler, D. M., Hartmann-Petersen, R., & Lindorff-Larsen, K. (2019). ***Biophysical and Mechanistic Models for Disease-Causing Protein Variants.*** Trends in Biochemical Sciences, 44(7), 575–588.  | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6579676/) | [Journal](https://doi.org/10.1016/j.tibs.2019.01.003)

Session 3.4 (Apr 15): Limitations of design
- Baker, D. (2019). ***What has de novo protein design taught us about protein folding and biophysics?*** Protein Science: A Publication of the Protein Society, 28(4), 678–683. | [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6423711/) | [Journal](https://doi.org/10.1002/pro.3588)

Session 3.5 (Apr 16): Teams present
- No paper
